### ISSUE TITLE
pypi dependencies list clarabel twice

### ISSUE THREAD

**Issue by adfea9c0** [cvxpy/cvxpy#2287]
**Describe the bug**
Looking at the pypi dependencies for 1.4, I see:

```bash
 $ curl https://pypi.org/pypi/cvxpy/1.4/json -H "Accept: application/json" | jq '.info.requires_dist'
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
100 25023  100 25023    0     0  2036k      0 --:--:-- --:--:-- --:--:-- 2036k
[
  "osqp >=0.6.2",
  "ecos >=2",
  "clarabel >=0.5.0",
  "scs >=3.0",
  "numpy >=1.15",
  "scipy >=1.1.0",
  "pybind11",
  "cylp >=0.91.5 ; extra == 'cbc'",
  "clarabel ; extra == 'clarabel'",
  "cvxopt ; extra == 'cvxopt'",
  "diffcp ; extra == 'diffcp'",
  "ortools <9.8,>=9.5 ; extra == 'glop'",
  "cvxopt ; extra == 'glpk'",
  "cvxopt ; extra == 'glpk_mi'",
  "gurobipy ; extra == 'gurobi'",
  "scipy >=1.6.1 ; extra == 'highs'",
  "Mosek ; extra == 'mosek'",
  "ortools <9.8,>=9.5 ; extra == 'pdlp'",
  "piqp ; extra == 'piqp'",
  "proxsuite ; extra == 'proxqp'",
  "PySCIPOpt ; extra == 'scip'",
  "scipy ; extra == 'scipy'",
  "setuptools >65.5.1 ; extra == 'scs'",
  "xpress ; extra == 'xpress'"
]
```

Clarabel is listed twice (`"clarabel >=0.5.0"` and `"clarabel ; extra == 'clarabel'"`) with the first inclusion being unconditional. Based on my reading of the website this is probably unintentional? Should clarabel be optional?

**Comment by SteveDiamond** [comment]
Clarabel should not be optional

**Comment by rileyjmurray** [comment]
That line in the docs (and perhaps a few others) definitely needs to be
updated.

**Comment by Transurgeon** [comment]
@adfea9c0 Thanks for pointing this out! I am currently working on a PR that tries to refactor the "advanced features" page. I will make sure to replace Clarabel instead of ECOS in the docs. 
Was there any other issues that you noticed? Any feedback can be helpful :)

**Comment by SteveDiamond** [comment]
Fixed in #2292
